src/ResourceManager/Network/Commands.Network/RouteTable/NewAzureRMRouteTableCommand.cs

```diff
@@ -70,6 +70,12 @@ public partial class NewAzureRmRouteTable : NetworkBaseCmdlet
             ValueFromPipelineByPropertyName = true)]
         public List<PSRoute> Route { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Disable BGP Route auto propagation.",
+            ValueFromPipelineByPropertyName = true)]
```

Comment:
@neaggarwMS once this is changed to a `SwitchParameter`, you can remove the `ValueFromPipelineByPropertyName` property